src/Proxy/StockTriggerLevelReportProxy.cs

```diff
@@ -45,7 +49,8 @@ and not (nvl(sl.location_id, -1) = nvl({locationId}, -1)
 
         public DataTable GetPartDataAtLocation(int locationId)
         {
-            var connection = new OracleConnection($"User Id = {this.userId}; Password = {this.password}; Data Source = {this.host};");
+            var dataSource = $"(DESCRIPTION=(ADDRESS=(PROTOCOL=TCP)(HOST={host})(PORT=1521))(CONNECT_DATA=(SERVICE_NAME={this.databaseName})(SERVER=dedicated)))";
```

Comment:
you could put this into another function if we're going to be newing up OracleConnections in more than one function